projects/packages/forms/src/contact-form/class-admin.php

```diff
@@ -704,17 +704,28 @@ public function grunion_manage_post_column_response( $post ) {
 		$post_content = get_post_field( 'post_content', $post->ID );
 		$content      = explode( '<!--more-->', $post_content );
 		$content      = str_ireplace( array( '<br />', ')</p>' ), '', $content[1] );
-		$chunks       = explode( "\nArray", $content );
-		if ( $chunks[1] ) {
-			// re-construct the array string
-			$array = 'Array' . $chunks[1];
-			// re-construct the array
-			$rearray         = Contact_Form_Plugin::reverse_that_print( $array, true );
+		$chunks       = explode( "\nJSON_DATA", $content );
+
+		$response_fields = array();
+
+		if ( is_array( $chunks ) ) {
```

Comment:
Should we perform a more strict check here? After all `explode( 'a', anything_here )` will result in an array as well. We need to make sure we're _parsing_ Feedback posts that meet the new format. `chunks[1]` would be a good starting point.

Reply:
Done. The code checks for array types and for feedback_id now.